pubspec.yaml

```diff
@@ -19,9 +19,9 @@ dependencies:
   mustache4dart: ^1.0.9
   package_config: '>=0.1.5 <2.0.0'
   path: ^1.3.0
-  quiver: '>=0.18.0 <0.25.0'
-  resource: '>=1.1.0 <3.0.0'
+  resource: ^1.1.0
```

Comment:
I'd just check that the sdk deps is bringing in a 2.0 version, and then bump our use up to ^2.0.0.